### ISSUE TITLE
Versioning and Repo support

### ISSUE THREAD

**Issue by Doomsdayrs** [zhkrb/cloudflare-scrape-Android#18]
1. https://jitpack.io | Jitpack takes github repos and auto builds jars for them to be distributed. 
A good example is one of my [dependencies](https://github.com/Doomsdayrs/shosetsu-services) for Shosetsu (that also plans to use your scraper). It looks like [this](https://jitpack.io/#Doomsdayrs/shosetsu-services) on jitpack

2. [Schematic versioning](https://semver.org/) is highly recommended for the above.
 In simple terms:
``(# of non-backwards compatible API changes).(# of feature addons).(# of bug fixes)`` Every time the a non backwards compatible API change is made, the number increments, and the proceeding numbers turn to 0. Same thing for feature addons and to bug fixes. 
My other project Jikan's [releases](https://github.com/Doomsdayrs/Jikan4java/releases) shows this well.

**Comment by zhkrb** [comment]
I have submitted two versions to jcenter for review, it is more convenient to use it.
Once approved, I will commit an update.

**Comment by zhkrb** [comment]
All have been updated.

**Comment by Doomsdayrs** [comment]
Thank you very much!
